src/types/FormTypes.ts

```diff
@@ -27,14 +27,35 @@ type TimelineForm = {
   synced: string;
   sessionType: string;
   reportLink: string;
-  reportSchedule: string;
+  repeatSchedule: string;
   startDate: string;
   endDate: string;
   startTime: string;
   endTime: string;
 };
 
-type MyForm = StudentForm | TestForm | TimelineForm;
-type MyFormKey = keyof StudentForm | keyof TestForm | keyof TimelineForm;
+type SessionType = {
+  number_of_fields_in_pop_form: number | null;
+  form_schema_id: number | null;
```

Comment:
where is form_schema_id in the notion note i shared ? it should be signup_form_name ... will check again

also where is session_type

please please cross check with notion doc

Reply:
If its being retreived from the database, it's `form_schema_id` only. The session table stored the reference.

Reply:
updated with singup_form_name now